src/js/hca/components/household-information/SpouseInformationSection.jsx

```diff
@@ -92,22 +93,20 @@ class SpouseInformationSection extends React.Component {
               ssn={this.props.data.spouseSocialSecurityNumber}
               onValueChange={(update) => {this.props.onStateChange('spouseSocialSecurityNumber', update);}}/>
 
-          <DateInput required
+          <ErrorableCurrentOrPastDate required
               label="Spouse’s date of birth"
               name="spouseBirth"
-              day={this.props.data.spouseDateOfBirth.day}
-              month={this.props.data.spouseDateOfBirth.month}
-              year={this.props.data.spouseDateOfBirth.year}
+              date={this.props.data.spouseDateOfBirth}
               onValueChange={(update) => {this.props.onStateChange('spouseDateOfBirth', update);}}/>
 
-          <DateInput required
-              errorMessage="Date of marriage cannot be before the Veteran's or the spouse's date of birth"
-              validation={isValidMarriageDate(this.props.data.dateOfMarriage, this.props.data.veteranDateOfBirth, this.props.data.spouseDateOfBirth)}
+          <ErrorableDate required
```

Comment:
Should this one be `ErrorableCurrentOrPastDate`, too? I think `ErrorableDate` will allow future dates.